rewrite-maven/src/main/java/org/openrewrite/maven/cache/RocksdbMavenPomCache.java

```diff
@@ -103,22 +105,23 @@ static synchronized RocksCache getCache(String workspace) {
     CacheResult<MavenMetadata> UNAVAILABLE_METADATA = new CacheResult<>(CacheResult.State.Unavailable, null);
     CacheResult<MavenRepository> UNAVAILABLE_REPOSITORY = new CacheResult<>(CacheResult.State.Unavailable, null);
 
-    public RocksdbMavenPomCache(@Nullable File workspace) {
+    public RocksdbMavenPomCache(@Nullable Path workspace) {
 
         assert workspace != null;
 
-        if(!workspace.exists() && !workspace.mkdirs()) {
-            throw new IllegalStateException("Unable to find or create maven pom cache at " + workspace);
-        } else if (!workspace.isDirectory()) {
+        File relativePath = new File( workspace + "/.pom");
```

Comment:
`relativePath` isn't a great name for this variable. It's likely to be an absolute (not that it matters particularly). A more descriptive and accurate name might be "pomCacheDir" or something along those lines.
```suggestion
        File pomCacheDir = new File( workspace.toFile(), ".pom");
```